co-authors-plus.php

```diff
@@ -725,10 +725,12 @@ function posts_where_filter( $where, $query ) {
 					$current_coauthor      = $this->get_coauthor_by( 'user_nicename', wp_get_current_user()->user_nicename );
 					$current_coauthor_term = $this->get_author_term( $current_coauthor );
 
-					$current_user_query  = $wpdb->term_taxonomy . '.taxonomy = \''. $this->coauthor_taxonomy.'\' AND '. $wpdb->term_taxonomy .'.term_id = \''. $current_coauthor_term->term_id .'\'';
-					$this->having_terms .= ' ' . $wpdb->term_taxonomy .'.term_id = \''. $current_coauthor_term->term_id .'\' OR ';
+					if ( $current_coauthor_term ) {
```

Comment:
One thing that came to my mind while reviewing this (And I'm just mentioning it for completeness) was: 
Could this ever be a "truthy" result and still not be what we're expecting?
From what I can tell it seems get_author_term() runs get_term_by() which does some checks against is_wp_error() so the code shouldn't ever return a wp_error which is good. The function declaration of get_term_by() does mention that it could return an array but that seems to only be if $output is ARRAY_(A|N) so it shouldn't be a problem.
All that to say that sometime just checking is something is "true" can lead to unexpected results. PHP's type system can lead to unexpected results http://php.net/manual/en/types.comparisons.php

One small thing I would change is that it seems like the indentation is being changed here. Probably a case of spaces vs tabs, depending on your IDE you should be able to set it to ensure it follows what the file is currently using.